exercises/knapsack/introduction.md

```diff
@@ -1,8 +1,10 @@
 # Introduction
 
-After months of careful planning, the expedition Bob works for is about to leave.
-As he is a sherpa, he will be paid the value he carried to the base camp.
+Lhakpa is a [Sherpa][sherpa] mountain guide and porter.
+After months of careful planning, the expedition Lhakpa works for is about to leave.
+She will be paid the value she carried to the base camp.
 
-In front of him are many items, each with a value and weight.
-Bob would gladly take all of the items, but his knapsack can only hold so much weight.
-Bob has to carefully consider which items to take so that the total value of his selection is maximized.
```

Comment:
I've just noticed that this last sentence was removed. Was that on purpose?

Reply:
Yes, it is mostly repeated in the first sentence of `instructions.md`: `Your task is to determine which items to take so that the total value of her selection is maximized, taking into account the knapsack's carrying capacity.`